base/src/commonMain/kotlin/utils/UnitSystem.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.base.utils
+
+enum class UnitSystem {
+
+    Metric, UK, US;
```

Comment:
`UK` and `US` can be named as `Mixed` (or `Both`) and `Imperial`

Reply:
I concur, thats a clearer naming

Reply:
Also Kotlin coding convections suggest naming enum parameters all upper case. I see that some classes like `DateFormatter` from `base` uses just the first letter upper case, but for example enum class `Style` from `alerts` uses it according to the convection. What is the kaluga preference for this?